packages/manager/modules/web-paas/src/index.less

```diff
@@ -0,0 +1 @@
+@import (less) 'ovh-ui-kit-bs/dist/css/oui-bs3.css';
```

Comment:
Could be imported directly in JS no ? Also maybe we should avoid this as it can have some weird side effects due to stylesheet rewriting and overriding, maybe we should leave the import to the app